spec/apiForSheetsSpec.js

```diff
@@ -3,6 +3,9 @@ const { createProbot } = require('probot');
 // The plugin refers to the actual app in index.js.
 const oppiaBot = require('../index');
 const apiForSheetsModule = require('../lib/apiForSheets');
+const checkPullRequestLabelsModule = require('../lib/checkPullRequestLabels');
+const checkPullRequestBranchModule = require('../lib/checkPullRequestBranch');
+const checkWIPModule = require('../lib/checkWipDraftPR');
```

Comment:
Suggest: ``checkWipModule`` (generally when handling camelcasing acronyms, it's conventional to treat it like a word and shorten the subsequent letters after the first letter).

Reply:
Done.